src/views/users/invite/EnrollApprovalView.tsx

```diff
@@ -30,6 +31,8 @@ interface WaitingListAccordionProps {
 export function EnrollApprovalView() {
   const [isApproved, setIsApproved] = useState(false);
   const [isRejected, setIsRejected] = useState(false);
+  const [isRejecting, setIsRejecting] = useState(false);
+  const [isApproving, setIsApproving] = useState(false);
```

Comment:
Could this be done as enum like?
For example 
`type SomeState = 'approved' | 'rejected' | 'rejecting' | 'approving'`

Reply:
That would be sane, though I don't have time to refactor the component for that for tomorrow

Reply:
ticket https://github.com/pvarki/rasenmaeher-ui/issues/18

Reply:
Alright, refactored to that. approvalState that uses those states. Also cleaned up the dialog to use that state